components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -960,6 +960,16 @@ int esp_websocket_client_send_bin(esp_websocket_client_handle_t client, const ch
     return esp_websocket_client_send_with_opcode(client, WS_TRANSPORT_OPCODES_BINARY, (const uint8_t *)data, len, timeout);
 }
 
+int esp_websocket_client_send_ping(esp_websocket_client_handle_t client, const char *data, int len, TickType_t timeout)
+{
+    return esp_websocket_client_send_with_opcode(client, WS_TRANSPORT_OPCODES_PING, (const uint8_t *)data, len, timeout);
```

Comment:
Wouldn't it be just easier to make `esp_websocket_client_send_with_opcode()` public?